src/lib/MedStickerEncryptionV2.js

```diff
@@ -0,0 +1,85 @@
+import create from "./factory";
+import { arrayBufferToString, encode } from "./utilities";
+import { CHARLIE } from "./constants";
+
+const scrypt = create("scrypt");
+const gcm = create("AES-GCM");
+
+/**
+ *
+ * @param secret
```

Comment:
For all these params please either add a description, or their type, or remove them. Right now they are not useful at all.